docs/general/mutations.md

```diff
@@ -134,3 +134,68 @@ type Mutation {
   updateFruitWeight(input: UpdateFruitWeightInput!): Fruit!
 }
 ```
+
+## Nested mutations
+
+To avoid a graph becoming too large and to improve discoverability, it can be helpful to group mutations in a namespace, as described by [Namespacing by separation of concerns](https://www.apollographql.com/docs/technotes/TN0012-namespacing-by-separation-of-concern/).
```

Comment:
nitpick: This PR adds a handful of links to the Apollo docs. We might want to make clear that these are thirdparty docs we're linking too to set the right expectations. Maybe by doing something like the following?

```suggestion
To avoid a graph becoming too large and to improve discoverability, it can be helpful to group mutations in a namespace, as described by [Apollos guide on Namespacing by separation of concerns](https://www.apollographql.com/docs/technotes/TN0012-namespacing-by-separation-of-concern/).
```

Reply:
Yep totally makes sense. Updated!

The graphql rules site is down for me right now so I couldn't find a "name" to ascribe the attribution to so I went with `[this community-created guide on GQL best practices](https://graphql-rules.com/rules/mutation-payload)`